HospitalToday/Program.cs

```diff
@@ -9,10 +10,24 @@ namespace HospitalToday
 {
     class Program
     {
+        static IService<Medicine> MedService;
+        static IService<Person> Doctors;
+        static IService<Person> Patients;
+        static IService<Report> Reports;
+
         static void Main(string[] args)
         {
-            IDoctorService doc = new DoctorService();
-            
+            InitServices();
+        }
+
+        static void InitServices()
```

Comment:
I think it is better to move InitServices() to specific static class.

Reply:
Ok, i created specific static class.